### PR TITLE
docs(ci): add deploy to heroku button

### LINKED ISSUE #119: Deploy on Heroku
Add a deploy to heroku button for the backend

https://devcenter.heroku.com/articles/heroku-button

### PR DESCRIPTION
**Pull request by kdsinghcoder**
# Issue Related with PR
- fixes #119 

# Checklist
- [x] Tests
- [x] Resolve Conflicts

# Screenshots

Original    |   Updated
:----------------: | :----------------:
image | image

### CODE REVIEW

**Review by YashKumarVerma — CHANGES_REQUESTED** [review]
please remove changes from yarn.lock file

**Review by kdsinghcoder — COMMENTED** [review]
Requested changes are implemented.

File: README.md [hunk 1]
```diff
@@ -1,6 +1,12 @@
...
-![Ensure Build](https://github.com/YashKumarVerma/authentico-backend/workflows/Ensure%20Build/badge.svg)
+# UniAuth Backend
+![GitHub issues](https://img.shields.io/github/issues/UniAuth/uniauth-backend)
+![GitHub pull requests](https://img.shields.io/github/issues-pr/Uniauth/uniauth-backend)
+![GitHub last commit](https://img.shields.io/github/last-commit/uniauth/uniauth-backend)
+![Gitlab code coverage](https://img.shields.io/gitlab/coverage/uniauth/uniauth-backend/master)
+![GitHub top language](https://img.shields.io/github/languages/top/uniauth/uniauth-backend)
+![Docker Automated build](https://img.shields.io/docker/automated/UniAuth/uniauth-backend)
+[![CodeFactor](https://www.codefactor.io/repository/github/uniauth/uniauth-backend/badge)](https://www.codefactor.io/repository/github/uniauth/uniauth-backend)
+![Ensure Build](https://github.com/UniAuth/uniauth-backend/workflows/Ensure%20Build/badge.svg)
```
Comment (YashKumarVerma) [inline README.md:9]: changes not of this PR.

where eis the deploy button?

**Review by YashKumarVerma — CHANGES_REQUESTED** [review]

**Review by kdsinghcoder — COMMENTED** [review]
Button added

File: README.md [hunk 2]
```diff
@@ -8,6 +8,8 @@
 [![CodeFactor](https://www.codefactor.io/repository/github/uniauth/uniauth-backend/badge)](https://www.codefactor.io/repository/github/uniauth/uniauth-backend)
 ![Ensure Build](https://github.com/UniAuth/uniauth-backend/workflows/Ensure%20Build/badge.svg)
 
+[![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy)
+
```
Comment (YashKumarVerma) [inline README.md:12]: this button does not automatically link this repo

**Review by YashKumarVerma — CHANGES_REQUESTED** [review]

**Review by kdsinghcoder — COMMENTED** [review]
@YashKumarVerma link updated

**Review by YashKumarVerma — APPROVED** [review]

**Comment by YashKumarVerma** [comment]
thanks !

**Pull request merged**